### PR TITLE
feat(account): ux improvement to import account functionality

### PR DESCRIPTION
**Pull request by vimercati-samir**
What this PR does 📖
UX improvement to import account functionality:
- auto completion when inserting single / multiple words 
- button inside input to clear input text
- button to reset the passphrase 
- message error when one of multiple words inserted is incorrect

Which issue(s) this PR fixes 🔨
[AP-2000](https://satellite-im.atlassian.net/browse/AP-2000)

Special notes for reviewers 🗒️

Additional comments 🎤

### CODE REVIEW

File: components/ui/TypeAhead/TypeAhead.html [hunk 1]
```diff
@@ -3,17 +3,26 @@
...
+    <input
+      v-model="searchText"
+      :class="`input is-${size} is-${type}`"
+      :placeholder="placeholder"
+      @mousedown="setFocus"
+      @input="update"
+      v-on:keyup.enter="onEnterPressed"
+      v-on:keyup.esc="lostFocus"
+      v-on:keydown.up="onUpBrowseItem"
+      v-on:keydown.down="onDownBrowseItem"
```
Comment (molimauro) [inline components/ui/TypeAhead/TypeAhead.html:16]: v-on => @

File: components/ui/TypeAhead/TypeAhead.vue [hunk 2]
```diff
@@ -92,12 +120,73 @@ export default Vue.extend({
...
+        let doesEveryOccurrenceMatch = true
+
+        for (const text of searchTextSplitted) {
+          if (
+            !suggestions.find((item) => item.suggestion === text.toLowerCase())
+          ) {
+            doesEveryOccurrenceMatch = false
+            break
+          }
+        }
```
Comment (molimauro) [inline components/ui/TypeAhead/TypeAhead.vue:155]: cant we use `.every`?

**Review by molimauro — CHANGES_REQUESTED** [review]
some minors

**Comment by molimauro** [comment]
image

A bit off on mobile :)

File: components/ui/TypeAhead/TypeAhead.vue (hunk 2 above)
Comment (molimauro) [reply components/ui/TypeAhead/TypeAhead.vue:155]: oh sorry didnt see the for, closing

File: pages/setup/ImportAccount/index.vue [hunk 3]
```diff
@@ -16,14 +16,22 @@ export default Vue.extend({
...
-  data(): { error: string; phrases: string[]; bipList: string[] } {
+  data(): { error: string; phrases: string[] } {
     return {
       error: '',
       phrases: [],
-      bipList: bip39.wordlists.english,
     }
   },
+  computed: {
+    bipList: function () {
```
[the remark this answers is not in the document]
Comment (vimercati-samir) [reply pages/setup/ImportAccount/index.vue]: resolved in commit 76e52f4e3781a7f02e9056aeda55d7da75ccfd7a

File: components/ui/TypeAhead/TypeAhead.html (hunk 1 above)
Comment (vimercati-samir) [reply components/ui/TypeAhead/TypeAhead.html]: resolved in commit 76e52f4e3781a7f02e9056aeda55d7da75ccfd7a

File: components/ui/TypeAhead/TypeAhead.html [hunk 4]
```diff
@@ -3,17 +3,26 @@
...
-    :class="`input is-${size} is-${type}`"
-    :placeholder="placeholder"
-    @mousedown="setFocus"
-    @input="update"
-    v-on:keyup.enter="onEnterPressed"
-    v-on:keyup.esc="lostFocus"
-    v-on:keydown.up="onUpBrowseItem"
-    v-on:keydown.down="onDownBrowseItem"
-  />
+  <div :class="{'search-box': true, 'right-icon': clearIcon && searchText.length > 0}">
```
[the remark this answers is not in the document]
Comment (vimercati-samir) [reply components/ui/TypeAhead/TypeAhead.html]: resolved in commit 76e52f4e3781a7f02e9056aeda55d7da75ccfd7a

**Comment by vimercati-samir** [comment]
> image
> 
> A bit off on mobile :)

resolved in commit 76e52f4e3781a7f02e9056aeda55d7da75ccfd7a

**Review by molimauro — APPROVED** [review]

**Pull request merged**